modules/images/webp-uploads/load.php

```diff
@@ -742,3 +742,33 @@ function webp_uploads_get_image_sizes_additional_mime_type_support() {
 
 	return $allowed_sizes;
 }
+
+/**
+ * Updates the quality of images used in the wp editor.
+ * For WP version less than 6.2 returns 82,
+ * Otherwise returns default quality.
+ *
+ * @since n.e.x.t
+ *
+ * @param int    $quality   Quality level between 1 (low) and 100 (high).
+ * @param string $mime_type Image mime type.
+ * @return int The updated quality for mime types.
+ */
+function webp_uploads_set_wp_image_editor_quality( $quality, $mime_type ) {
+	global $wp_version;
+
+	$valid_mime_transforms = webp_uploads_get_upload_image_mime_transforms();
+
+	// Return early for non image or invalid mime types.
+	if ( ! isset( $valid_mime_transforms[ $mime_type ] ) ) {
+		return $quality;
+	}
+
+	if ( version_compare( '6.2', $wp_version, '>=' ) ) {
+		return $quality;
+	}
+
+	// Default quality set to 82 for all mimes for WP version less than 6.2.
+	return 82;
```

Comment:
The code here is not doing what the comment says:
* The code returns the default quality for 6.2 and below, and 82 for a version greater than 6.2.
* The comment says to return the default quality for 6.2 and above, and 82 for a version lower than 6.2.

In any case, please review the solution proposed in the issue:
* Below 6.1, we should universally return 82.
* In 6.1+, we should return 82 only for `image/webp`, and the default quality otherwise.

Reply:
New changes reflect these suggestions. Please have a look.